src/pages/Progress/Progress.test.tsx

```diff
@@ -66,8 +66,8 @@ test("display member is not found when query parameter is not passed", () => {
   });
   render(<Progress />);
 
-  const errorMessage = screen.getByText("Member not found!");
-  expect(errorMessage).toBeInTheDocument();
+  const header = screen.getByRole("heading", { level: 1 });
```

Comment:
`header`라고 하면 좀 오해의 소지가 있을 것 같습니다.

```suggestion
  const heading = screen.getByRole("heading", { level: 1 });
```

Reply:
오호 여기까지 수정해서 commit 정리후 merge할게요!